crates/astria-sequencer/src/bridge/bridge_lock_action.rs

```diff
@@ -99,7 +103,10 @@ impl ActionHandler for BridgeLockAction {
             .await
             .wrap_err("failed to get byte cost multiplier")?;
         let fee = byte_cost_multiplier
-            .saturating_mul(get_deposit_byte_len(&deposit))
+            .saturating_mul(
+                calculate_base_deposit_fee(&deposit)
```

Comment:
Because we are doing saturating arithmetic here I would just keep doing that. Instead of returning early just replace it with with u128::MAX

Reply:
Replaced with `unwrap_or(u128::MAX)` in 3faab50. Let me know if this works!